build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -150,6 +151,13 @@ jobs:
         #Note: The project qualifier is always required when using a feed name. Also, do not use organization scoped feeds.
         publishVstsFeed: 'ProjectReunion/Project.Reunion.nuget.internal'
 
+  - ${{ if eq(parameters.publishToMaestro, 'true') }}:
+    - template: ..\..\eng\common\Maestro-PublishBuildToMaestro.yml
+      parameters:
+        IsGitHubRepo: true
+        AssetName: 'Microsoft.ProjectReunion.Foundation.TransportPackage'
+        AssetVersion: '$(version).${{ parameters.prereleaseVersionTag }}'
```

Comment:
>${{ parameters.prereleaseVersionTag }} [](http://example.com/codeflow?start=34&length=38)

do we get a "." at end if prereleaseVersionTag is missing?

Reply:
The precondition is that there will always be a prereleaseVersionTag :)

Reply:
Refactring this because '$(version).${{ parameters.prereleaseVersionTag }}' is repeated else where

Reply:
even for a stable release build?